lms/templates/revisions/dashboard_revision_results.underscore

```diff
@@ -0,0 +1,16 @@
+<span class="title edit-name">
+    <% if (totalCount > 0 ) { %>
+        <%= interpolate(
+            ngettext(
+                "You have (%(num_items)s) pending revision:",
+                "You have (%(num_items)s) pending revisions:",
+                totalCount
+            ),
+            { num_items: totalCount },
+            true
+        ) %>
+    <% } else { %>
+        <%= gettext("You have no pending revisions.") %>
+    <% } %>
+</span>
+<ul class="revision-result-list"></ul>
```

Comment:
@bdero This can just be `revision-list` I think.

Also, please rename this file to `dashboard_revision_list.underscore`.